docusaurus/docs/dev-docs/file-structure.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Project Structure
+displayed_sidebar: devDocsSidebar
+---
+
```

Comment:
the file changed to project-structure for docu: https://www.notion.so/strapi/Project-structure-7670f4ece82649b59f4c335f2456a0e8